packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/mentionIntegration.tests.js

```diff
@@ -641,5 +644,36 @@ export default function() {
             this.$element.find('p').first().text('@');
             this.clock.tick(10);
         });
+
+        test('aria-activedescendant should exist on textbox when mention pops up', function(assert) {
+            const done = assert.async();
+            const valueChangeSpy = sinon.spy(() => {
+                const $content = this.$element.find(`.${HTML_EDITOR_CONTENT}`);
+                if(valueChangeSpy.calledOnce) {
+                    this.clock.tick(10);
+                    const $items = this.getItems();
+                    const $focusedItem = $items.filter(`.${FOCUSED_STATE_CLASS}`).first();
+
+                    assert.strictEqual($focusedItem.attr('id'), $content.attr('aria-activedescendant'), 'textbox element aria id should match active item id when mentions pops up');
```

Comment:
The first parameter is an actual value and the second parameter is an expected one.
```suggestion
                    assert.strictEqual($content.attr('aria-activedescendant'), $focusedItem.attr('id'), 'textbox element aria id should match active item id when mentions pops up');
```